worlds/ffmq/Items.py

```diff
@@ -0,0 +1,295 @@
+from BaseClasses import ItemClassification, Item
+
+fillers = {"Cure Potion": 61, "Heal Potion": 52, "Refresher": 17, "Seed": 2, "Bomb Refill": 19,
+           "Projectile Refill": 50}
+
+
+class ItemData:
+    def __init__(self, item_id, classification, groups=[], data_name=None):
```

Comment:
if `groups` is ever modified, it'll modify it for every `ItemData` that doesn't explicitly define this. A tuple would be safer.